doc/new-tracing/tracers_doc_generated.md

```diff
@@ -5,7 +5,7 @@
 ### [Trace Messages](#trace-messages)
 
 1. __BlockFetch__
-	1. __Client__
+	1. __Clientⓣ__
```

Comment:
Some strange unicode symbols added, here and below.

Reply:
\9443
\9442 
\9436